Disallow initiating a new Remote Access session when in Secure Mode (#17968)
nvaccess/nvda: user_docs/en/userGuide.md

<!-- KC:beginInclude -->
| Name |Key |Description|
|---|---|---|

> | Connect or disconnect | `NVDA+alt+r` | If a remote session is in progress, disconnects from it. Otherwise, starts a new Remote session. Only works if currently connected in [secure mode](#SecureMode). |

Reviewer: Shouldn't this work regardless of secure mode?

Author: This was poor wording on my part. It works regardless of secure mode, but in secure mode it only works if disconnecting. How about wording like
```suggestion
| Connect or disconnect | `NVDA+alt+r` | If a remote Access session is in progress, disconnects from it. Otherwise, starts a new session. In [secure mode](#SecureMode), this gesture only works to disconnect from an existing session. |
```
Or we could just disregard the note, as it can be taken as read that in secure mode this only disconnects a session, since creating new sessions is not permitted in secure mode.
```suggestion
| Connect or disconnect | `NVDA+alt+r` | If a remote session is in progress, disconnects from it. Otherwise, starts a new Remote session. |
```

Reviewer: My preference is to disregard the note.

```suggestion
| Connect or disconnect | `NVDA+alt+r` | If a Remote Access session is in progress, disconnects from it. Otherwise, starts a new session. |
```
